src/extern.jl

```diff
@@ -2,21 +2,23 @@
 
 # Main function
 """
-    heppcat(Y,k,iters;init=homppca(Y,k))
+    heppcat(Y, k, iters; init=homppca(Y,k), vknown=false, eps=0)
 
 Estimate probabilistic PCA model for noise that is heteroscedastic across samples.
 
-Inputs are:
+# Inputs:
 + `Y` : list of matrices (each column is a sample)
 + `k` : number of factors
 + `iters` : number of iterations to run
 + `init`  : initial model (will be modified in-place)
++ `vknown::Bool` : variances are known (do not update) default `false`
++ 'eps::Real' : epsilon to avoid variances going towards zero, useful if normal method fails, default = 0
 Output is a [`HePPCATModel`](@ref) object.
 """
-function heppcat(Y,k,iters;init=homppca(Y,k))
+function heppcat(Y, k, iters::Int; init=homppca(Y,k), vknown::Bool=false, eps::Real=0)
```

Comment:
```suggestion
function heppcat(Y, k, iters::Int; init=homppca(Y,k), vknown::Bool=false, vmin::Real=zero(eltype(Y)))
```
I hope to support units some day, so we should use `zero` not `0`.

Reply:
Agreed, using `zero` should help make things more type-stable.

I think that handling units will take a bit more thought though - the variances should have units that are the square of `Y`. Let's leave that for a future PR. :)